package.json

```diff
@@ -79,7 +79,7 @@
     "formidable": "1.1.1",
     "fs-extra": "5.0.0",
     "glob": "7.1.2",
-    "gulp": "3.9.1",
+    "gulp": "^3.9.1",
```

Comment:
Why this change?

Reply:
oh this was not intentional but the current state of this file has this change as well. I am guessing it was from running yarn.

Reply:
actually im wrong. reverting file.